src/app.py

```diff
@@ -0,0 +1,48 @@
+"""
+An example stored procedure. __main__ provides an entrypoint for local development
+and testing.
+"""
+
+from snowflake.snowpark.session import Session
+from snowflake.snowpark.dataframe import col, DataFrame
+from snowflake.snowpark.functions import udf
+from src.functions import combine
+
+def run(snowpark_session: Session) -> DataFrame:
+    """
+    A sample stored procedure which creates a small DataFrame, prints it to the
+    console, and returns the number of rows in the table.
+    """
+
+    combine_udf = udf(combine)
+
+    schema = ["col_1", "col_2"]
+
+    data = [
+        ("Welcome to ", "Snowflake!"),
+        ("Learn more: ", "https://www.snowflake.com/snowpark/"),
+    ]
+
+    df = snowpark_session.create_dataframe(data, schema)
+
+    df2 = df.select(combine_udf(col("col_1"), col("col_2")).as_("hello_world")).sort(
+        "hello_world", ascending=False
+    )
+
+    return df2
+
+
+if __name__ == "__main__":
+    # This entrypoint is used for local development (`$ python src/procs/app.py`)
+
+    from src.util.local import get_env_var_config
+
+    print("Creating session...")
+    session = Session.builder.configs(get_env_var_config()).create()
+    session.add_import("src/functions.py", 'src.functions')
```

Comment:
I would recommend using somethign like:
```
import functions and then
functions.__file__
``` 

mostly because that makes the code lets susceptible to path changes. However for that to work well you need to properly setup PYTHONPATH that is why the suggestions in settings.json

Reply:
Fixed, ran locally fine. Going through PR checks now